internal/security/secretstore/init.go

```diff
@@ -363,35 +363,40 @@ func (b *Bootstrap) BootstrapHandler(ctx context.Context, _ *sync.WaitGroup, _ s
 		os.Exit(1)
 	}
 
-	cert := NewCerts(req, configuration.SecretService.CertPath, rootToken, configuration.SecretService.GetSecretSvcBaseURL(), lc)
-	existing, err := cert.AlreadyinStore()
-	if err != nil {
-		lc.Error(err.Error())
-		os.Exit(1)
-	}
+	// With Vault moving to non-TLS mode, the cert related logic is obsolete
+	if configuration.SecretService.CertPath != "" {
```

Comment:
Are we going to still support TLS mode if configured or simplify the code to No TLS. @bnevis-i  thoughts?
If we are never going to use TLS, then we should remove the code totally.

Reply:
Given the subject of distributing microservices across the wire keeps coming up, I am hesitant to go all in.